src/Response.php

```diff
@@ -6,4 +6,7 @@
 
 class Response
 {
+    public function __construct(private int $status, private mixed $body, private array $headers)
```

Comment:
```suggestion
    public function __construct(
        private int $status, 
        private mixed $body, 
        private array $headers
    )
```
*For consistent styling*